pipeline_dp/private_beam.py

```diff
@@ -308,3 +315,117 @@ def __init__(self, fn: Callable, label: Optional[str] = None):
     def expand(self, pcol: pvalue.PCollection):
         return pcol | "flatten values" >> beam.ParDo(
             FlatMap._FlattenValues(map_fn=self._fn))
+
+
+class PrivateCombineFn(beam.CombineFn):
+
+    @abc.abstractmethod
+    def add_private_input(self, accumulator, input):
```

Comment:
As this is developer-facing, could we add some docs: what is this method supposed to do?

add_private_input implies that the input variable should already be protected by some privacy, or should this be just "add_input"?